src/app/round.ts

```diff
@@ -7,6 +7,7 @@ import { TimePeriod } from './time-period';
  */
 export interface FirebaseRound {
   flowerSpeciesIds: string[];
+  beeSpeciesIds?: string[];
```

Comment:
Why are we adding the array of bee species IDs to the firebase round?

Reply:
Oh, good catch!

Reply:
I think this was a temporary measure to set the bees until the templates are added.